src/app/pages/send/send-crypto-asset-preview/components/send-crypto-asset-details.tsx

```diff
@@ -0,0 +1,80 @@
+import { useMemo } from 'react';
+
+import { StacksTransaction, TokenTransferPayload, addressToString } from '@stacks/transactions';
+import { Box, Stack, Text } from '@stacks/ui';
+import { microStxToStx, truncateMiddle } from '@stacks/ui-utils';
+import BigNumber from 'bignumber.js';
+
+import { STX_DECIMALS } from '@shared/constants';
+import { CryptoCurrencies } from '@shared/models/currencies.model';
+import { createMoney } from '@shared/models/money.model';
+
+import { useConvertCryptoCurrencyToFiatAmount } from '@app/common/hooks/use-convert-to-fiat-amount';
+import { convertAmountToBaseUnit } from '@app/common/money/calculate-money';
+import { TransactionFee } from '@app/components/fee-row/components/transaction-fee';
+import { SpaceBetween } from '@app/components/space-between';
+import { Caption } from '@app/components/typography';
+
+function getFeeWithDefaultOfZero(tx?: StacksTransaction) {
+  if (!tx) return createMoney(0, 'STX');
+  const fee = microStxToStx(Number(tx.auth.spendingCondition.fee));
+  return createMoney(new BigNumber(fee), 'STX');
+}
+
+interface SendCryptoAssetDetailsProps {
+  feeCurrency: CryptoCurrencies;
+  symbol: string;
+  token: string;
+  unsignedTx: StacksTransaction;
+}
+// TODO: Handle bitcoin transactions
```

Comment:
When doing this, think of how we can compose to keep stx/btc/others as isolated as possible